src/java/org/apache/cassandra/tools/nodetool/stats/StatsTable.java

```diff
@@ -82,4 +82,15 @@ public class StatsTable
     public double localReadWriteRatio;
     public Long twcsDurationInMillis;
     public String twcs;
+
+    public double QueryLatencyMs;
+    public double PostFilteringReadLatencyms;
+    public String DiskUsedBytes;
+    public double SSTableIndexesHit;
+    public double IndexSegmentsHit;
+    public double RowsFiltered;
+    public long TotalQueryTimeouts;
+    public int TotalIndexCount;
+    public int TotalQueryableIndexCount;
+    public String TotalQueryableIndexRatio;
```

Comment:
nit/style: For the above, make sure to start field names w/ lower-case letter

Reply:
...or we could just prefix them all with `sai`, which I was going to suggest anyway, given some of them are pretty general, like disk used bytes.

Reply:
Added prefix as sai.